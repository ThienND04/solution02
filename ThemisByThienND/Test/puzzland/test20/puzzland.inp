7
10 31
A J R G W T P I C H 
G P
P I
G W
J W
G I
T H
W H
J G
W I
J C
J R
T P
R G
A T
W P
A I
J P
J T
I C
R I
R C
J H
A R
G H
G T
A W
P H
C H
T C
R W
R H
7 14
N B Y J H Z P 
N P
Y H
N Y
J H
Y Z
J Z
Y P
Z P
H Z
H P
N J
N H
B H
N B
7 12
W P S A N L M 
A M
S L
W L
W N
W M
L M
W S
N L
P S
P L
A N
S A
10 37
M H A U R L O J P F 
H F
L O
M P
A L
A R
L J
R O
U R
A O
H J
R P
R J
A J
M R
J P
M F
M J
H P
J F
M L
M O
O F
H A
H R
P F
M H
L P
H U
O P
U P
A U
U O
R F
M A
A F
M U
O J
4 5
R M W B 
R B
R M
W B
R W
M B
8 24
G Q A F O L I K 
F K
G K
Q L
L I
A L
G Q
G O
G A
O K
A K
A F
A I
F I
Q A
L K
G L
Q O
Q K
F O
Q I
O L
F L
I K
A O
2 1
V I 
V I
