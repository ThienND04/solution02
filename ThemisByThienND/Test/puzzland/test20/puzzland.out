AICJGRWPTH
NBHJZYP
WNASPLM
MAHJLOPURF
IMPOSSIBLE
GAFILOQK
VI
